apps/frontend/components/EditorHeader.tsx

```diff
@@ -90,16 +101,20 @@ export default function Editor({
     loading && submissionId ? 500 : null
   )
 
-  const { showSignIn } = useAuthModalStore((state) => state)
   useEffect(() => {
     auth().then((session) => {
       if (!session) {
-        toast.info('Log in to use submission & auto save feature')
+        toast.info('Log in to use submission & save feature')
+      } else {
+        setUserName(session.user.username)
+        getLocalstorageCode()
```

Comment:
```suggestion
        setUserName(session.user.username)
```

유저네임 바뀌면 아래쪽에 있는 `useEffect`에서 `getLocalstorageCode` 함수 호출하니까 없어도 될 것 같아요!